letter.md

```diff
@@ -1,25 +1,21 @@
-The Comprehensive R Archive Network (CRAN), is unusual among user-contributed software package infrastructure due to the high standards to which contributions are held, and the continuous checking system that validates packages against reverse dependencies and operating systems. 
+The Comprehensive R Archive Network (CRAN) is exceptional infrastructure. Thanks to high standards for contributions and a thorough continuous checking system, users can trust that any package from CRAN is likely to work, be compatible with other CRAN packages, and leave the local R package library in a consistent state. These guarantees are fundamental to the success and popularity of R, and they would not be possible without the tireless effort and diligence of the CRAN team.
 
-This combination of features allows users to trust that any software they install from CRAN is likely to work, be compatible with other CRAN packages, and leave their local R package library in a consistent state. The great effort undertaken to move these software package pain points out of the way of statistics practitioners has made a large contribution toward R becoming an extremely popular statistical programming environment.
+Because of its success, CRAN has become a fundamental global resource, not only for independent researchers, but also for large organizations across academia, government, and industry, all of whom all depend on the ability to contribute packages in a reliable and predictable manner. The scope and the need have grown bigger than one individual or small team, and it is appropriate that the R community exercise a voice with respect to the conduct and administration of CRAN.
```

Comment:
Great angle.

This is ambiguous to me:

"The scope and the need have grown bigger than one individual or small team, and it is appropriate..."

Are we saying the scope has grown bigger than then CRAN team, or the scope has grown bigger than it was under a different type of user base?

Reply:
I edited the second paragraph, hopefully it is a bit clearer now. I am trying to explain a basic assumption about fairness, not sure if it came across 100%.